harvester/aws-container-definitions.json

```diff
@@ -28,9 +23,62 @@
             "options": {
                 "awslogs-group": "/ecs/harvester",
                 "awslogs-region": "eu-central-1",
-                "awslogs-stream-prefix": "${version}",
+                "awslogs-stream-prefix": "harvester-${version}",
                 "awslogs-multiline-pattern": "^\\[?\\d\\d\\d\\d\\-\\d\\d\\-\\d\\d \\d\\d:\\d\\d:\\d\\d,\\d\\d\\d"
             }
         }
+    },
+    {
+      "name": "celery-beat-container",
+      "image": "${REPOSITORY}/harvester:${version}",
+      "cpu": 0,
+      "essential": true,
+      "command": ["celery", "-A", "harvester", "beat", "-s", "/tmp/celerybeat-schedule"],
+      "environment": [
+        { "name" : "PYTHONUNBUFFERED", "value" : "1" },
+        { "name" : "APPLICATION_MODE", "value" : "${mode}" },
+        { "name" : "POL_DJANGO_DEBUG", "value" : "0" },
+        { "name" : "POL_SECRETS_SURFCONEXT_SECRET_KEY", "value" : "not-used" },
```

Comment:
This is old school I think. Should work without it

Reply:
Ah, yeah probably. I will remove it.